spyder/widgets/tests/data/ham.erl

```diff
@@ -0,0 +1,10 @@
+ham(N) -> ham(N, 1, 1).
```

Comment:
Adding a new file type requires changing `setup.py` and `MANIFEST.in` because we package our tests and all their supporting files as part of Spyder.

So please use a file extension that we already support, such as `.txt`.